README.md

```diff
@@ -1 +1,49 @@
-# portfolio-mobile-first
\ No newline at end of file
+
+## Prerequisites
+Install code editor to be able to run the commands in the terminal. VSCode
+
+## Built With
+HTML and CSS
+
+## Setup
+download the linters folder and copy it to your project You can clone the project through git clone https://github.com/eyoab01/portfolio-mobile-first
+
+## Install
+Install the linters folder to your project. Install the fontawesome for icons in the project 
+
+## Usage
+To get a local copy up and running follow these simple steps-->download the linters folder and copy it to your project You can clone the project through git clone https://github.com/eyoab01/portfolio-mobile-first
+
+## Run tests
+To run tests, run the following command:
+
+## Deployment
+My portifolio website can be viewed at:  [https://github.com/eyoab01/portfolio-mobile-first]
+
+## Future Features
+ navbar and headline sections. 
+
+## Live Demo
+# https://github.com/eyoab01/portfolio-mobile-first
+
+## Authors
+👤 Tesfaye Teklu Feyissa
+
+GitHub: @eyoab01
+Twitter: @tesfayeteklu10
+LinkedIn: www.linkedin.com/in/tesfaye-teklu-352067105
+
+## License
+This project is MIT licensed.
+Feel free to check the issues page.
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the issues page.
+
+## Show your support
+If you like this project you can support me.
+
+## Acknowledgments
+
+I would like to thank my team mates at microverse for their support.
```

Comment:
- [ ] Seem you omitted a lot of headings from the README. I'll outline them below in the order in which you're to include them. please use the [REAME TEMPLATE](https://github.com/microverseinc/readme-template) Provided to complete all sections exactly . 

## Sections in order:
[📖 About the Project]project)
[🛠 Built With](https://github.com/microverseinc/readme-template#built-with)
[Tech Stack](https://github.com/microverseinc/readme-template#tech-stack)
[Key Features](https://github.com/microverseinc/readme-template#key-features)
[🚀 Live Demo](https://github.com/microverseinc/readme-template#live-demo)
[💻 Getting Started](https://github.com/microverseinc/readme-template#getting-started)
[Setup](https://github.com/microverseinc/readme-template#setup)
[Prerequisites](https://github.com/microverseinc/readme-template#prerequisites)
[Install](https://github.com/microverseinc/readme-template#install)
[Usage](https://github.com/microverseinc/readme-template#usage)
[Run tests](https://github.com/microverseinc/readme-template#run-tests)
[Deployment](https://github.com/microverseinc/readme-template#triangular_flag_on_post-deployment)
[👥 Authors](https://github.com/microverseinc/readme-template#authors)
[🔭 Future Features](https://github.com/microverseinc/readme-template#future-features)
[🤝 Contributing](https://github.com/microverseinc/readme-template#contributing)
[⭐️ Show your support](https://github.com/microverseinc/readme-template#support)
[🙏 Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements)
[❓ FAQ (OPTIONAL)](https://github.com/microverseinc/readme-template#faq)
[📝 License](https://github.com/microverseinc/readme-template#license)

The linked will take you to all the paths. please fill them in your repo, to make it professional

Reply:
i have finished the project but my status on student dashboard for this project is changed to incomplete, i can't perform pull request. what can i do please?